stories/expenses/ExpenseFormItems.stories.mdx

```diff
@@ -28,7 +28,7 @@ import { webpackCollective } from '../mocks/collectives';
   <Story name="Invoice type (file not required)">
     {() => {
       return (
-        <Formik initialValues={{ type: 'INVOICE', currency: 'EUR' }}>
+        <Formik initialValues={{ type: 'INVOICE', currency: 'EUR', items: [{ id: '1', incurredAt: '2020-02-07' }] }}>
```

Comment:
To make sure we have a fixed date:

| Before | After |
| --- | --- |
| ![](https://lp-00001-a8c7a.fra1.digitaloceanspaces.com/cli4e0w6q2kjzmo0el8bu56i1/079eb2cc377099db1ade037860870e3ae879e504a0a71c81e915b0e398799d76.png) | ![](https://lp-00001-a8c7a.fra1.digitaloceanspaces.com/cli4e0w6q2kjzmo0el8bu56i1/ce6043950e87d0f5203336564c409b68e3fb38f50a1638c2c89fee2254f12001.png) |
</details>